data/json/bionics.json

```diff
@@ -1181,9 +1177,9 @@
   {
     "id": "bio_syringe",
     "type": "bionic",
-    "name": { "str": "Intravenous Needletip" },
-    "description": "The flesh of the inside of your middle finger has been replaced with a tube leading into your arteries, with a retractable needle extending out of the tip of the finger when clenched.  This lets you pump intravenous drugs like morphine directly into your bloodstream without needing to carry a syringe.",
-    "occupied_bodyparts": [ [ "hand_l", 1 ] ]
+    "name": { "str": "Intravenous Injector" },
+    "description": "You have a drug injector surgically mounted in your torso, leading into your arteries.  This lets you pump intravenous drugs like morphine directly into your bloodstream without needing to carry a syringe.",
```

Comment:
How do you load stuff into it?

Reply:
Just like with the previous version.
I imagine this CBM as a tube leading from a finger/some point on the torso to a vein.

Reply:
It would appear a bit difficult to maneuver a needle/tube sticking out of your torso (to insert into the vial/pouch/whatever of IV drug).

Reply:
Now it uses two slots in the left arm.

Reply:
Two?